### PR TITLE
update Black with line 120

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

seems that the black config was a dummy one with lots of not-relevant settings to this project

Moreover changing the line length to 120 as it is already with flake8 and it common standard now

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -44,7 +44,6 @@ repos:
     hooks:
       - id: black
         name: Black code
-        args: ["-S"]
```
Comment (Borda) [inline .pre-commit-config.yaml:47]: This has been already in the config as `skip-string-normalization = true`

**Comment by willingc** [comment]
@Borda If we use Ruff, I believe that we can remove black.

**Comment by Borda** [comment]
> @Borda If we use Ruff, I believe that we can remove black.

tbh, never tried but still, I may tend to keep it... of have the same setting? :)

**Comment by willingc** [comment]
Let's do this @Borda. I'll go ahead and merge this since I am a fan of 120.

FYI on the differences between Ruff and Black for formatting. They are fairly subtle at the moment. https://docs.astral.sh/ruff/formatter/#the-ruff-formatter

**Review by willingc — APPROVED** [review]

**Pull request merged**